.github/workflows/check_release_name_bump.yml

```diff
@@ -0,0 +1,35 @@
+name: Check release bump
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: get latest-release
+      id: last_release
+      uses: InsonusK/get-latest-release@v1.0.1
```

Comment:
Did you manage to find an official action to get releases?

Reply:
Unfortunately not, all the github actions regarding grabbing the latest release i found are in the market place like this one